src/packages/core/protocol/account.ts

```diff
@@ -5,7 +5,16 @@ import Hex from 'crypto-js/enc-hex';
 import Base64 from 'crypto-js/enc-base64';
 
 import * as secp256k1 from 'secp256k1';
-import { getPrivateKeyBuffer, getXOR, sha256, ripemd160 } from '../utils';
+import { getPrivateKeyBuffer, getXOR, sha256, ripemd160, isValidAddress } from '../utils';
+
+
+import { IAPIConfig } from '..';
+import { BalanceApi } from '../api/balance';
+
+const APIServerTestNet: IAPIConfig = {
```

Comment:
we already have a default configuration in the config folder. Remove this one and let the user pass a config object as optional last argument of our new methods.

Reply:
Removed